rocAL/rocAL/source/meta_data/caffe2_meta_data_reader_detection.cpp

```diff
@@ -179,11 +179,10 @@ void Caffe2MetaDataReaderDetection::read_lmdb_record(std::string file_name, uint
                 for (int i = 0; i < boundBox_size / 4; i++)
```

Comment:
// Parsing width of image
            img_size.w= image_proto.dims(0);
            // Parsing height of image
            img_size.h = image_proto.dims(1);

            Remove these unused variables